### PR TITLE
Reorg packages (3)

### PR DESCRIPTION
**Pull request by mbtools**
Ref #6686

### CODE REVIEW

File: src/repo/stage/package.devc.xml [hunk 1]
```diff
@@ -1,9 +1,9 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (larshp) [inline src/repo/stage/package.devc.xml:1]: byte order mark removed?

**Comment by mbtools** [comment]
Using github.dev messed up eol and boms. VScode got it right 🤷

**Comment by larshp** [comment]
> Using github.dev messed up eol and boms. VScode got it right 🤷

need to add a abaplint check for it sometime

**Review by larshp — APPROVED** [review]
pulled ok

image

**Pull request merged**